### PR TITLE
sig-api-machinery: add clientgofix repo

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://github.com/kubernetes/org/issues/1731

/assign @deads2k @lavalamp @fedebongio 
/hold
for sign off from leads ~+ for repo migration~

### CODE REVIEW

File: sig-api-machinery/README.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ The following [subprojects][subproject-definition] are owned by sig-api-machiner
   - https://raw.githubusercontent.com/kubernetes-client/python-base/master/OWNERS
   - https://raw.githubusercontent.com/kubernetes-client/python/master/OWNERS
   - https://raw.githubusercontent.com/kubernetes-client/ruby/master/OWNERS
+  - https://raw.githubusercontent.com/kubernetes-sigs/clientgofix/master/OWNERS
```
Comment (nikhita) [inline sig-api-machinery/README.md]: Repo will be migrated once it's moved to a dummy org, figured I'd get this PR out though

Comment (nikhita) [reply sig-api-machinery/README.md]: The repo exists now - https://github.com/kubernetes-sigs/clientgofix

**Comment by deads2k** [comment]
Yes, we take ownership for this.

/lgtm
/approve

**Pull request merged**